integration-tests/gradle/gradle.properties

```diff
@@ -0,0 +1 @@
+version = 999-SNAPSHOT
```

Comment:
@gsmet here is another version property to keep in mind when releasing.

Reply:
That's basically another `gradle.properties` file with `version` property, in case you have a script for that.

Reply:
Thanks for thinking of it :). I'll adjust the release script right away.